x-pack/legacy/plugins/ml/public/application/datavisualizer/file_based/components/filebeat_config_flyout/filebeat_config.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FindFileStructureResponse } from '../../../../../../common/types/file_datavisualizer';
+
+export function createFilebeatConfig(
+  index: string,
+  results: FindFileStructureResponse,
+  ingestPipelineId: string,
+  username: string | null
+) {
+  return [
+    'filebeat.inputs:',
+    '- type: log',
+    '  paths:',
+    "  - '<add path to your files here>'",
```

Comment:
Is it worth internationalizing this bit of text?

Reply:
Fixed in 91ad89dbdc4c02f8a32936b83e1ce549208ee886